interface.go

```diff
@@ -1,26 +1,12 @@
 package discovery
 
-import (
-	"context"
-	"time"
+import moved "github.com/libp2p/go-libp2p-core/discovery"
 
-	pstore "github.com/libp2p/go-libp2p-peerstore"
-)
+// Deprecated: use skel.Advertiser instead.
```

Comment:
what the hell is `skel`?

Reply:
Used to mean "skeleton", this is outdated now.